File: Python/BOJ/1120 문자열.py
import sys
A, B = sys.stdin.readline().strip().split()
n, m = len(A), len(B)
result = int(10e9)
for i in range(m-n+1):
    temp = 0
    for j in range(n):
        print(A[j], B[i+j])
        if A[j] != B[i+j]:
            temp += 1
    result = min(result, temp)
print(result)

File: Java/BOJ/1120 문자열.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        String A = st.nextToken();
        String B = st.nextToken();
        int n = A.length();
        int m = B.length();
        int result = Integer.MAX_VALUE;
        for(int i = 0; i < m-n+1; i++) {
        	int temp = 0;
        	for (int j = 0; j < n; j++) {
        		if(A.charAt(j) != B.charAt(i+j)) {
        			temp++;
        		}
        	}
        	result = Math.min(result, temp);
        }
        System.out.println(result);
    }
}